src/gui/transferlistsortmodel.cpp

```diff
@@ -164,29 +166,33 @@ bool TransferListSortModel::lessThan(const QModelIndex &left, const QModelIndex
             return !invalidL;
         }
     }
-    else if (column == TorrentModel::TR_LAST_ACTIVITY) {
-        const qlonglong vL = left.data().toLongLong();
-        const qlonglong vR = right.data().toLongLong();
+
+    case TorrentModel::TR_LAST_ACTIVITY: {
+        qlonglong vL = left.data().toLongLong();
+        qlonglong vR = right.data().toLongLong();
```

Comment:
I was thinking all cases should follow the same pattern: have `const` or not.
I will move this change to another commit.